packages/eslint-config/package.json

```diff
@@ -26,7 +26,9 @@
   ],
   "main": "index.js",
   "dependencies": {
-    "@madebywild/eslint-config-typescript": "^0.2.3"
+    "@madebywild/eslint-config-typescript": "^0.1.0",
+    "eslint-config-prettier": "^8.3.0",
```

Comment:
same of https://github.com/madebywild/eslint-config/pull/3/files/161a990a67f655b5b89e26095599d29aa3e882bb..49858376ee32a25e390dbfa58bb3fa5838a3d467#r685208094